src/pages/GenericPage.js

```diff
@@ -64,4 +64,9 @@ GenericPage.propTypes = {
   finalisableDataType: PropTypes.string,
   refreshData: PropTypes.func.isRequired,
   topRoute: PropTypes.bool,
+  isDataCircular: PropTypes.bool,
+};
+
+GenericPage.defaultProps = {
+  isDataCircular: true,
```

Comment:
Thats a pretty specific name that doesn't convey the difference very well. I'd just go straight `useDeepEquals` or `useShallowEquals`, depending on what you made the default in `react-native-generic-table-page`

Reply:
Talked about it. We'll just deal with it for now (i.e. never going to be changed).